.github/workflows/test.yml

```diff
@@ -24,16 +24,20 @@ jobs:
       fail-fast: false
       matrix:
         os: [ubuntu-20.04, ubuntu-22.04, actuated-arm64-6cpu-8gb]
-        go-version: [1.20.x, 1.21.x]
+        go-version: [1.21.x, 1.22.x]
         rootless: ["rootless", ""]
         race: ["-race", ""]
         criu: ["", "criu-dev"]
         dmz: ["", "runc_nodmz"]
         exclude:
+          # Go 1.22 + glibc < 2.32 + nsenter does not play well (issue 4233),
+          # thus nsenter package won't compile. Disable ubuntu 20.04 + go 1.22.
```

Comment:
Maybe add "TODO: enable this again when Go 1.22.4 is available"

Reply:
done, thanks!